sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1614,6 +1614,10 @@ public void setSampleAssociations(ContactReferenceDto sourceContact, CaseReferen
 					sampleFacade.cloneSampleForCase(sample, caze);
 				}
 			});
+
+			// The samples for case are not persisted yet, so use the samples from contact since they are the same
+			Date untilDate = service.computeFollowUpuntilDate(caze, contact.getSamples());
```

Comment:
Remove the unnecessary variable declaration and pass `computeFollowUpuntilDate` directly to `setFollowUpUntil`